cli/readme.md

```diff
@@ -0,0 +1,39 @@
+# Raml CLI
+
+### Building project
+
+This is the raml generator CLI addition
+
+* in order to build it run `./gradlew distZip -p cli` at the root project to generate the application archive
+
+* you will find a cli.zip generated which contain the generated program
+
+* if you run ./cli generate help you will get the cli manual
+```
+NAME
+        cli generate - generate raml stub
+
+SYNOPSIS
+        vrap generate [-out <outputFolder>] [-p <basePackageName>] [--]
```

Comment:
I think it instead of `vrap` it should be `cli`?

Reply:
Yep, it is a typo there 😅